src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private int capacity;
+
+    private String description;
+
+    public CinemaHall() {
```

Comment:
Don't create a default constructor when it is not needed. If Hibernate requires a no-arg constructor, it can be package-private to restrict unnecessary usage from outside the package.